src/crdt.rs

```diff
@@ -134,6 +133,17 @@ impl ReplicatedData {
             repair_addr,
         )
     }
+    pub fn new_entry_point(gossip_addr: SocketAddr) -> Self {
```

Comment:
What do you mean by `entry_point` here? Would `new_from_base_addr` make more sense?

Reply:
By entry point I meant that it’s a gossip network entry point, it’s a dummy entry that had an invalid public key.  It’s used to attach to the network initially and will eventually get purged

Reply:
Can you add code comments? You don't add a test in this PR and the only explanation is in the PR description and the review comment above.

Reply:
Nevermind, I just didn't grok the test.